LiskHQ/lisk-sdk
# Fix loader expectation and add configs - Closes #3591

### What was the problem?
Response was expecting wrong format

### How did I fix it?
Update to extract correct format

### How to test it?
Sync with the testnet, and observe it has correct response.

```
console.log(result)
process.exit(0)
```
put above after
https://github.com/LiskHQ/lisk-sdk/pull/3592/files#diff-a37a28379e9954d0ebb1d0c3f44784bfL208

### Review checklist

* The PR resolves #3591 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
